src/pytest_markdown_docs/plugin.py

```diff
@@ -204,7 +204,15 @@ def find_object_tests_recursive(
 
 class MarkdownDocstringCodeModule(pytest.Module):
     def collect(self):
-        module = import_path(self.path, root=self.config.rootpath)
+        if pytest.version_tuple >= (8, 1, 0):
+            # consider_namespace_packages is a required keyword argument in pytest 8.1.0
+            module = import_path(
+                self.path, root=self.config.rootpath, consider_namespace_packages=True
```

Comment:
Not actually sure what the implications of consider_namespace_packages would be here